### PR TITLE
my 2c...

### PR DESCRIPTION
**Pull request by JedMeister**
I had a quick look at your code. This is what I'd do... It doesn't all work yet but the structure is there. I have made RT the doc root so you no longer need all the other stuff. The Mahara appliance is based on LAPP so that might be of some use in regards to ideas of how stuff should/could be done...

Personally this is how I like to do it: Log into TKLDev in an SSH session and build the appliance (using CHROOT_ONLY=y).

Once it has built open another SSH session (to TKLDev). In one fab-chroot in (to build/root.sandbox) and start postgres & apache and play with the setup. In the other session adjust your build code as you go...

### CODE REVIEW

**Comment by DocCyblade** [comment]
Looking at the code now, I like your two ssh sessions method. Thanks for the kick start in the right direction. If I have any questions about the code I'll post them here

File: conf.d/main [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/bin/sh -ex
```
Comment (DocCyblade) [inline conf.d/main:1]: I see you pulled out all the conf.d scripts and put most of the "stuff" in the main. Reason for this? Also see you changed from bash to sh, reason behind that. Just learning you know.

**Comment by JedMeister** [comment]
I saw your comment but I didn't seem to be able to reply there so am doing it here.

> I see you pulled out all the conf.d scripts and put most of the "stuff" in the main. Reason for this?

No _really_ good reason... Just trying to maintain consistency with the other appliances. 

Also it was easier as I robbed some stuff from another appliance's conf.d main file. I copy/pasted from your config into conf.d/main...

**Comment by DocCyblade** [comment]
Ok, just thought I would ask. Going to work on this, this weekend.

**Comment by JedMeister** [comment]
Cool! :+1:

**Pull request merged**